src/ConvertToSarif/ConvertOptions.cs

```diff
@@ -43,9 +43,10 @@ internal class ConvertOptions
 
         [Option(
             'a',
-            "plugin-assembly-path",
+            "plugin-assembly-paths",
             Default = null,
-            HelpText = "Path to plugin assembly containing converter types.")]
-        public string PluginAssemblyPath { get; internal set; }
+            HelpText = "Comma-separated list of paths to plugin assemblies containing converter types.",
+            Separator = ',')]
+        public string[] PluginAssemblyPaths { get; internal set; }
```

Comment:
As we discussed, removed this capability.

---
In reply to: [120984734](https://github.com/Microsoft/sarif-sdk/pull/582#discussion_r120984734) [](ancestors = 120984734)